src/char/int_party.cpp

```diff
@@ -923,20 +919,21 @@ int inter_party_CharOffline(uint32 char_id, int party_id) {
 		}
 	}
 
-	if(!p->party.count)
-		//Parties don't have any data that needs be saved at this point... so just remove it from memory.
-		idb_remove(party_db_, party_id);
+	// Parties don't have any data that needs be saved at this point... so just remove it from memory.
+	if( !p->party.count ){
```

Comment:
```suggestion
	if( p->party.count == 0 ){
```
For readability since it's not a bool type.